tomviz/python/Recon_DFT.py

```diff
@@ -33,17 +31,12 @@ def transform_scalars(self, dataset):
         self.progress.message = 'Initialization'
         Nz = Ny
         w = np.zeros((Nx, Ny, Nz // 2 + 1)) #store weighting factors
-        v = pyfftw.n_byte_align_empty(
-            (Nx, Ny, Nz // 2 + 1), 16, dtype='complex128')
-        v = np.zeros(v.shape) + 1j * np.zeros(v.shape)
-        recon = pyfftw.n_byte_align_empty(
-            (Nx, Ny, Nz), 16, dtype='float64', order='F')
-        recon_fftw_object = pyfftw.FFTW(
-            v, recon, direction='FFTW_BACKWARD', axes=(0, 1, 2))
+        v = pyfftw.empty_aligned(
+            (Nx, Ny, Nz // 2 + 1), dtype='complex64', n=8)
 
-        p = pyfftw.n_byte_align_empty((Nx, Npad), 16, dtype='float64')
-        pF = pyfftw.n_byte_align_empty(
-            (Nx, Npad // 2 + 1), 16, dtype='complex128')
+        p = pyfftw.empty_aligned((Nx, Npad), dtype='float32', n=8)
```

Comment:
Why choose 8 as the alignment here?

Reply:
_n_ is the number of bytes for this object and I want to reduce the memory consumption in half.

Reply:
Doesn't ```n``` determine the byte alignment?

Reply:
Ah, yes you're right. I misinterpreted it. Let's keep n at 16 then.